tests/EntgeltUtils/test_app.py

```diff
@@ -0,0 +1,126 @@
+import os
+from boto3 import client
+import pytest
+from moto import mock_s3
+import pandas as pd
+
+from EntgeltUtils.app import App
+from finanzen_base.Utils.extract_filename import extract_filename
+
+
+@pytest.fixture
+def aws_credentials():
+    """Mocked AWS Credentials for moto."""
+    os.environ["AWS_ACCESS_KEY_ID"] = "testing"
+    os.environ["AWS_SECRET_ACCESS_KEY"] = "testing"
+    os.environ["AWS_SECURITY_TOKEN"] = "testing"
+    os.environ["AWS_SESSION_TOKEN"] = "testing"
+
+
+@pytest.fixture
+def s3_client(aws_credentials):
+    with mock_s3():
+        conn = client("s3", region_name="us-east-1")
+        yield conn
+
+
+@pytest.fixture
+def bucket_name():
+    return "my-test-bucket"
+
+
+@pytest.fixture
+def s3_test(s3_client, bucket_name):
+    s3_client.create_bucket(Bucket=bucket_name)
+    yield
+
+
+def test_app_init(s3_client, s3_test, bucket_name):
```

Comment:
```suggestion
def test_init_app(s3_client, s3_test, bucket_name):
```

Naming convention für Funktion ist Verb verwenden